plugin/vscode/extension.js

```diff
@@ -373,15 +374,22 @@ async function activateAsync() {
 
   toDispose.push(
     vscode.workspace.onDidChangeTextDocument((event) => {
-      logAsyncErrors(
-        (async () => {
-          if (isLintable(event.document)) {
-            await linters
-              .getLinter(event.document)
-              .textChangedAsync(event.contentChanges);
-          }
-        })()
-      );
+      /*
+      Event fires twice on first character input event of the open file. 
+      First one is caused by dirty-state change and the secibd is the actual character change.
+      Below is the implementation of a suggested workaround by maintainer: https://github.com/Microsoft/vscode/issues/50344
```

Comment:
> Below is the implementation
What is this comment referring to? The next line? The next 10 lines? I suggest being more descriptive. Perhaps something like this:
```js
// comment, yada yada. talks about isBogusEvent.
let isBogusEvent = event.contentChanges.length === 0;
if (!isBogusEvent) {
  ...
}
```